src/main/java/seedu/address/model/company/Date.java

```diff
@@ -30,7 +32,7 @@ public Date(String date) {
      * Constructs an empty {@code Date}
      */
     public Date() {
-        this.date = null;
+        this.date = LocalDate.parse(DEFAULT_DATE_STRING);
```

Comment:
If using the singleton pattern, this.date should point to the DEFAULT_DATE

Reply:
done

Reply:
resolved